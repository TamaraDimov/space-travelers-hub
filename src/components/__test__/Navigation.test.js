import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Navbar from '../Navigation';

test('Navbar renders correctly', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Navbar />
    </MemoryRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
