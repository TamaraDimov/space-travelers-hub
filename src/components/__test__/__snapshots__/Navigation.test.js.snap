// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div>
  <nav
    className="navContainer"
  >
    <div
      className="title"
    >
      <img
        alt="Space Travel Hub logo"
        className="logo"
        src="space.png"
      />
      <h1
        className="navHeader"
      >
        Space Traveler's Hub
      </h1>
    </div>
    <ul
      className="navLinks"
    >
      <li>
        <a
          className="/navRockets"
          href="/rockets"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className="navMissions"
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li
        className="navBorderLine"
      />
      <li>
        <a
          className="navMyProfile"
          href="/myProfile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
